addons/attachment_indexation/models/ir_attachment.py

```diff
@@ -4,15 +4,22 @@
 import logging
 import xml.dom.minidom
 import zipfile
-from pdfminer.pdfinterp import PDFResourceManager, PDFPageInterpreter
-from pdfminer.converter import TextConverter
-from pdfminer.pdfpage import PDFPage
 
 from odoo import api, models
 
 _logger = logging.getLogger(__name__)
+
+try:
+    from pdfminer.pdfinterp import PDFResourceManager, PDFPageInterpreter
+    from pdfminer.converter import TextConverter
+    from pdfminer.pdfpage import PDFPage
+except ImportError:
+    PDFResourceManager = PDFPageInterpreter = TextConverter = PDFPage = None
+    _logger.warning("Attachement indexation of PDF documents is unavailable because the 'pdfminer' Python library cannot be found on the system.")
```

Comment:
If I may suggest, it's extra nice if we suggest a reference or command that can be used to install the optional dependency :-)

```suggestion
    _logger.warning("Attachment indexation of PDF documents is unavailable because the 'pdfminer' Python library cannot be found on the system. You may install it from https://pypi.org/project/pdfminer.six/ (e.g. `pip3 install pdfminer.six`)")
```